openapi.json

```diff
@@ -5972,7 +5983,7 @@
           "NPM"
         ],
         "summary": "Get issues by package",
-        "description": "Get all the issues related with a particular npm package version.\nThis endpoint returns the issue type, location, and additional details related to each issue in the `props` attribute.\n\nYou can [see here](https://socket.dev/npm/issue) the full list of issues.",
+        "description": "Get all the issues related with a particular npm package version.\nThis endpoint returns the issue type, location, and additional details related to each issue in the `props` attribute.\n\nYou can [see here](https://socket.dev/npm/issue) the full list of issues.\n        \n        This endpoint consumes 1 unit of your quota.",
```

Comment:
@alxhotel I think we missed stripping the indent here 🙂

Reply:
Found the bug! I made a PR https://github.com/SocketDev/depscan/pull/3525

Reply:
@alxhotel Did we get it released?

Reply:
Its doesn't seem like it

Reply:
@voxpelli Yay! It's now updated!